retroshare-gui/src/gui/settings/ServerPage.cpp

```diff
@@ -62,11 +62,11 @@
 ///
 
 // Tabs numbers *after* non relevant tabs are removed. So do not use them to add/remove tabs!!
-const static uint32_t TAB_HIDDEN_SERVICE_OUTGOING = 0;
+//const static uint32_t TAB_HIDDEN_SERVICE_OUTGOING = 0;
```

Comment:
Please keep these. It prevents the next code cleaning to actually rename them from 0 to n-1 and therefore cause bugs in the addressing of tabs.

Reply:
Comment left to not doing that.
And this not prevents to add a new intermediate Tab in ui file, like it happens sometimes.